package/kedro_viz/api/rest/router.py

```diff
@@ -49,6 +50,18 @@ async def get_single_node_metadata(node_id: str):
     return get_node_metadata_response(node_id)
 
 
+@router.post("/preferences")
```

Comment:
Any reason why we created a new api endpoint for setting preferences instead of using existing api/nodes/ by passing user preferences set in redux state to api/nodes/